yarn_api_client/application_master.py

```diff
@@ -19,13 +19,15 @@ class ApplicationMaster(BaseYarnAPI):
     :param int port: Proxy HTTP port
     :param int timeout: API connection timeout in seconds
     :param boolean kerberos_enabled: Flag identifying is Kerberos Security has been enabled for YARN
+    :param boolean https: Flag identifying HTTPS is used instead of HTTP for YARN
+    :param boolean https_verify: Flag identifying SSL certificate should be ignored in case of HTTPS
```

Comment:
As we are forwarding this parameter all way to `requests`, could you please use/append similar documentation as the one available in requests: `Either a boolean, in which case it controls whether we verify the server’s TLS certificate, or a string, in which case it must be a path to a CA bundle to use`